ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template2.kt

```diff
@@ -1,8 +1,6 @@
-package com.revenuecat.purchases.ui.revenuecatui.templates.template2
+package com.revenuecat.purchases.ui.revenuecatui.templates
```

Comment:
Moved this to the parent package since we haven't extracted any composables for `Template2` yet. If/when we start extracting composables specifically for different templates, we can create those folders.